classes/class-uagb-rest-api.php

```diff
@@ -303,6 +303,28 @@ public function blocks_register_rest_fields() {
 		 * @return array
 		 */
 		public function get_editor_store_initial_state() {
+			$initial_state = array();
+		
+			$add_gbs = $this->get_gbs_initial_states();
+			if ( ! empty( $add_gbs ) ) {
+				$initial_state = $add_gbs;
+			}
+
+			return $initial_state;
```

Comment:
```suggestion
			return $this->get_gbs_initial_states();
```

can't we do this.

Whats the reason of createaing a new function.

Reply:
Just for keeping API callback function but now removed.